mm-bot/Makefile

```diff
@@ -0,0 +1,35 @@
+# Database variables
+container = postgres
+user = user
+password = 123123123
```

Comment:
Can't we do this differently?

Reply:
What do you mean? 
These are the default values only for local environment using docker containers. In production we use postgres without docker
Also, when you use `make create_db`, you can override all these values.